client/dashboard/sites/backups/index.tsx

```diff
@@ -50,6 +59,40 @@ export function SiteBackupsCallout( {
 	);
 }
 
+function Backups( { site }: { site: Site } ) {
+	const [ view, setView ] = useState< View >( {
+		type: 'table',
+		fields: [ 'date', 'action', 'user' ],
+	} );
+
+	const { data: activityLog, isLoading: isLoadingActivityLog } = useQuery(
+		siteLastRewindableActivityLogEntriesQuery( site.ID )
```

Comment:
Hmmm.... how does the pagination work if we never pass anything to the endpoint 🙈 currently the endpoint in the data layer always pass `number: 20`.

I guess the endpoint should return all backups and then we do pagination from frontend via `filterSortAndPaginate()` (?)